tests/jballerina-unit-test/src/test/resources/test-src/documentation/markdown_function.bal

```diff
@@ -52,3 +52,7 @@ function searchPeople () returns [string, int, float] {
 # + y - another thing to be added
 # + return - the sum of them
 function add (int x, int y) returns int  { return x + y; }
+
+# Test variable `'testQuotedConst`
```

Comment:
We need to give a valid reference in order to test the removal of the quote here. Since `'testQuotedConst` is not defined in the scope this line anyway will give a warning.